### PR TITLE
Updated to have java config examples

### PR DESCRIPTION
**Pull request by kirksc1**
This commit updates the scalability.adoc to have java configuration examples for the first 2 snips in the doc.

### CODE REVIEW

File: spring-batch-docs/asciidoc/scalability.adoc [hunk 1]
```diff
@@ -34,7 +34,10 @@ First, we review the single-process options. Then we review the multi-process op
...
 
 The simplest way to start parallel processing is to add a `TaskExecutor` to your Step
-configuration. For example, you might add an attribute of the `tasklet`, as shown in the
+configuration.
+
+[role="xmlContent"]
+For example, you might add an attribute of the `tasklet`, as shown in the
 following example:
 
 [source, xml]
```
Comment (mminella) [inline spring-batch-docs/asciidoc/scalability.adoc]: Needs a `role="xmlContent"`

File: spring-batch-docs/asciidoc/scalability.adoc [hunk 2]
```diff
@@ -57,8 +83,10 @@ Note that this means there is no fixed order for the items to be processed, and
...
 addition to any limits placed by the task executor (such as whether it is backed by a
 thread pool), there is a throttle limit in the tasklet configuration which defaults to 4.
-You may need to increase this to ensure that a thread pool is fully utilized, as shown in
-the following example:
+You may need to increase this to ensure that a thread pool is fully utilized.
+
+[role="xmlContent"]
+For example you might increase threads, as shown in the following example:
 
 [source, xml]
```
Comment (mminella) [inline spring-batch-docs/asciidoc/scalability.adoc]: Needs a `role="xmlContent"`

**Comment by kirksc1** [comment]
Yes, I will take a shot at finishing out the file.  We'll see how far I can get.

**Comment by kirksc1** [comment]
I couldn't figure out how to translate the last XML config to java config since MultiResourceItemReader doesn't appear to have a "resource" property.

**Comment by cppwfs** [comment]
@kirksc1 it does have a resource`s` property though.  https://github.com/spring-projects/spring-batch/blob/master/spring-batch-infrastructure/src/main/java/org/springframework/batch/item/file/MultiResourceItemReader.java#L242   If you would like to update the XML.

**Comment by mminella** [comment]
Thanks for the updates!  Merged and polished as c3814cc27da02f3a51679ffca6918213e3dbace7

**Pull request closed without merging**